vip-helpers/vip-utils.php

```diff
@@ -1640,3 +1640,19 @@ function vip_get_hyper_servers( $hyperdb, $operation = 'all', $dataset = 'global
 
 	return $hyperdb->hyper_servers[ $dataset ];
 }
+
+/**
+ * Checks if a given domain is a VIP convenience domain. Examples of a VIP convenience
+ * domain are `example.go-vip.co` and `example.go-vip.net`.
+ *
+ * @param string $domain Domain to check
+ * @return bool True if the domain is a convenience domain
+ */
+function is_vip_convenience_domain( string $domain ) {
+	if ( '' === $domain ) {
+		return false;
+	}
```

Comment:
What's the advantage of breaking this bit out?

Reply:
faster by 0.000000001 seconds duh